### ISSUE TITLE
Support for parsing custom corpus

### ISSUE THREAD

**Issue by oveddan** [adeshpande3/Facebook-Messenger-Bot#47]
It would be great if you supported parsing a custom corpus that a user can provide, that is already formatted in an argument that is recognizable, and can feed into `createDataset.py` 
For example, if you could pass a text file that looks like:

message: How's it going today?
response: It's going alright
message: What's for dinner tonight?
response: Chicken baked with cheese

**Comment by adeshpande3** [comment]
Yeah totally agree with that. Things kinda busy for me on the life end, but PRs are welcome 🙏

**Comment by caraneel** [comment]
On this note-- is there any way to see what the format is of the document that is being fed to createDataset.py? I realize the actual files contain personal info, I just want to know the structure of it so I can recreate it. The fbchat-archive-parser doesn't work for my message data, so I want to re-create the file that would have resulted from running fbcap ./messages.htm > fbMessages.txt

**Comment by adeshpande3** [comment]
Actually now that I think back to this project (it's been a while for me), I think the fbMessages.txt file is actually pretty similar to the format @oveddan was talking about (correct me if I'm wrong though). https://github.com/adeshpande3/Facebook-Messenger-Bot/issues/28 Basically just `username: message` on each line and then you should enter your `username` here (https://github.com/adeshpande3/Facebook-Messenger-Bot/blob/master/createDataset.py#L7)

**Comment by oveddan** [comment]
ok so what you're saying is, it expects a file in the format of:

somePerson: hi how's it going
someOtherPerson: ok
somePerson: what's for dinner tonight?
someOtherPerson: chicken on rice

Would it work if given a file like this?  It would be great if there a sample file format for fb messages somewhere in this repo, considering the fb message gathering repo is obsolete.
